pkg/iac-providers/terraform/v14/testdata/tfjson/recursive-loop-duplicate-locals.json

```diff
@@ -0,0 +1,20 @@
+{
+    "aws_iam_user": [
+      {
+        "id": "aws_iam_user.lb",
+        "name": "lb",
+        "module_name": "dummy",
+        "source": "dummy/main.tf",
+        "plan_root": "./",
+        "line": 15,
+        "type": "aws_iam_user",
+        "config": {
+          "name": "joe != \"\" ? \"joe-fred\" : fred",
+          "tags": "${merge(${merge({\n          CreatedBy = \"Terraform\"\n        }, {\"this\":\"that\"})},\n  {\n    Name = joe != \"\" ? \"joe-fred\" : fred\n  })}"
```

Comment:
can you change it to how you changed this part https://github.com/accurics/terrascan/pull/900/files#diff-9e7e2f86151ba69659926d94f991b79ee8c8e03d62733d1aa2dd72207485282aR406 ?

Reply:
That is not possible because here it is a string with an expression that needs to be evaluated. Once we add support for functions this will also change.